.testcaferc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "browsers": ["chrome"],
+  "src": ["./__tests__/*.test.js"],
+  "proxyByPass": ["localhost:3000"],
+  "clientScripts": [
+    { "module": "@testing-library/dom/dist/@testing-library/dom.umd.js" }
+  ],
+  "developmentMode": true
```

Comment:
Is this necessary?

Reply:
per : https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#developmentmode

No needs, it's only for debugging with TestCafe team

Reply:
We should also remove it from https://github.com/MyCryptoHQ/MyCrypto